src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_NUMBER = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int RANDOM_SIDE = 10;
+    private ColorSupplier supplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRundomFigure() {
+        String color = supplier.getRandomColor();
+        int randomFigure = random.nextInt(FIGURE_NUMBER);
+        int randomSide = getRandomSide();
+        return switch (randomFigure) {
+            case 0 -> new Circle(randomSide, color);
+            case 1 -> new IsoscelesTrapezoid(randomSide, randomSide, randomSide, color);
+            case 2 -> new Rectangle(randomSide, randomSide, color);
+            case 3 -> new RightTriangle(randomSide, randomSide, color);
+            case 4 -> new Square(randomSide, color);
+            default -> new Square(randomSide, color);
+        };
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULT_RADIUS, Color.WHITE.name());

```

Comment:
When creating the default figure, consider passing the `Color.WHITE` enum directly instead of using `Color.WHITE.name()` to maintain consistency with the use of enums.